static/app/utils/performanceForSentry.tsx

```diff
@@ -417,3 +417,32 @@ export const addExtraMeasurements = (transaction: TransactionEvent) => {
     // Defensive catch since this code is auxiliary.
   }
 };
+
+/**
+ * A util function to help create some broad buckets to group entity counts without exploding cardinality.
+ *
+ * @param tagName - Name for the tag, will create `<tagName>` in data and `<tagname>.grouped` as a tag
+ * @param max - The approximate maximum value for the tag, A bucket between max and Infinity is also captured so it's fine if it's not precise, the data won't be entirely lost.
+ * @param n - The value to be grouped, should represent `n` entities.
+ * @param [buckets=[1,2,5]] - An optional param to specify the bucket progression. Default is 1,2,5 (10,20,50 etc).
+ */
+export const setGroupedEntityTag = (
+  tagName: string,
+  max: number,
+  n: number,
+  buckets = [1, 2, 5]
+) => {
```

Comment:
I feel like the API here is a little spooky! Asking for a `max` before the `n` is pretty confusing, and there's no default `max`. Plus, I feel like allowing people to pass custom buckets will allow inconsistent tags, but maybe that's fine!